src/model.js

```diff
@@ -0,0 +1,242 @@
+const yelp = require("yelp-fusion");
+const proj4 = require("proj4");
+const _ = require("lodash");
+const async = require("async");
+
+const {
+  koopProviderYelp: { api_key },
+} = require("config");
+const client = yelp.client(api_key);
+const YELP_API_MAX_RADIUS = 40000;
+
+const proj = proj4("GOOGLE", "WGS84");
+
+function Model() {}
+
+// each model should have a getData() function to fetch the geo data
+// and format it into a geojson
+Model.prototype.getData = function (req, callback) {
+  if (req.query && req.query.returnCountOnly) {
+    return callback(null, {
+      type: "FeatureCollection",
+      count: 2001,
+      features: [],
+    });
+  }
+  const queries = buildQueries(req.query);
+  const featureCollection = {
+    type: "FeatureCollection",
+    features: [],
+  };
+
+  async.each(queries, search, finish);
```

Comment:
Why use async.each?  It looks like you `queries` is always a one element array.

Reply:
similar to above. will remove.